xml/operations-configuring-identity-websso.xml

```diff
@@ -356,7 +358,7 @@ keystone_sp_conf:
         description: This is the ADFS identity provider.
     mapping:
         id: mapping1
-        rules_file: adfs_mapping.json
+        rules_file: /var/lib/ardana/openstack/my_cloud/config/adfs_mapping.json
```

Comment:
Shouldn't there be a keystone folder in the path?

Reply:
ahhh good catch!